shell/common/shell.cc

```diff
@@ -373,6 +374,10 @@ Shell::Shell(DartVMRef vm, TaskRunners task_runners, Settings settings)
           task_runners_.GetUITaskRunner(),
           std::bind(&Shell::OnServiceProtocolGetDisplayRefreshRate, this,
                     std::placeholders::_1, std::placeholders::_2)};
+  service_protocol_handlers_[ServiceProtocol::kGetSkSLsExtensionName] = {
+      task_runners_.GetUITaskRunner(),
```

Comment:
Nit: This doesn't have to be serviced on the UI task runner right (the writes are already atomic)? We can perform these operations on the IO thread. This will give fewer opportunities to block the UI thread. Granted its just `debug` and `profile` modes, but still.